module-4/backend/week_one.md

```diff
@@ -5,18 +5,47 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+ --Learning about `this` was super important.
+ 
 2. What are some tools to help debug JavaScript code?
+ --The `debugger` and just go ahead and `console.log()` everything!
+ 
 3. What are some tools you need in order to unit test your JavaScript?
+ --Mocha and Chai, and now Selenium.
+ 
 4. What is the syntax for invoking a function?
+ --Don't forget the parentheses after the name. Pass in any parameters you'll need.
+ 
 5. What is CORS?
+ --Cross Origin Resource Sharing. I think it's referencing the use of resources from other domains. Grabbing stuff to bring render on the DOM.
+ 
 6. How do we enable CORS?
+ --I don't know off the top of my head.
```

Comment:
Server-side config will set this up to restrict requests to API. It's enabled automatically client-side by Chrome